compiler/src/iree/compiler/Pipelines/Options.cpp

```diff
@@ -46,6 +46,7 @@ void InputDialectOptions::bindOptions(OptionsBinder &binder) {
 #ifdef IREE_COMPILER_PLUGIN_HAVE_STATIC_INPUT_STABLEHLO
           "  =stablehlo     - Legalize from StableHLO ops.\n"
           "  =stablehlo_xla - Legalize from StableHLO ops (with XLA cleanup preprocessing).\n"
+          "  =vhlo          - Legalize from VHLO ops (VHLO is translated to StableHLO).\n"
```

Comment:
Could we make it so that vhlo deserialization runs automatically in both the stablehlo and stablehlo_xla pipeline? As is, the _xla variant seems not supported.